src/org/openbravo/erpCommon/ad_callouts/SE_Order_Organization.java

```diff
@@ -53,42 +58,48 @@ protected void execute(CalloutInfo info) throws ServletException {
         FieldProvider[] td = null;
 
         final String strOrgId = info.getStringParameter("inpadOrgId", IsIDFilter.instance);
-        final String strBPartnerId = info.getStringParameter("inpcBpartnerId",
-                IsIDFilter.instance);
-        final String strBPartnerLocationId = info.getStringParameter(
-                "inpcBpartnerLocationId", IsIDFilter.instance);
 
-        info.addResult("inpiscashvat",
-                CashVATUtil.isCashVAT(strinpissotrx, strOrgId, strBPartnerId,
-                        strBPartnerLocationId));
+        OBCriteria<OrgWarehouse> orgWarehouseCriteria = OBDal.getInstance().createCriteria(OrgWarehouse.class);
+        orgWarehouseCriteria.add(Restrictions.eq(OrgWarehouse.PROPERTY_ORGANIZATION, OBDal.getInstance().get(Organization.class, strOrgId)));
+        orgWarehouseCriteria.add(Restrictions.eq(OrgWarehouse.PROPERTY_ACTIVE, true));
+        orgWarehouseCriteria.setProjection(Projections.property(OrgWarehouse.PROPERTY_WAREHOUSE));
 
+        List<String> warehouseIds = new ArrayList<>();
+        for (Object obj : orgWarehouseCriteria.list()) {
+            Warehouse warehouse = (Warehouse) obj;
+            warehouseIds.add(warehouse.getId());
+        }
         try {
             ComboTableData comboTableData = new ComboTableData(info.vars, this, "TABLE",
                     "M_Warehouse_ID", "197",
                     strinpissotrx.equals("Y") ? "C4053C0CD3DC420A9924F24FC1F860A0" : "",
-                    Utility.getReferenceableOrg(info.vars,
-                            info.vars.getStringParameter("inpadOrgId")),
+                    Utility.getReferenceableOrg(info.vars, strOrgId),
                     Utility.getContext(this, info.vars, "#User_Client", info.getWindowId()),
                     0);
             Utility.fillSQLParameters(this, info.vars, null, comboTableData,
                     info.getWindowId(), "");
             td = comboTableData.select(false);
-        } catch (Exception ex) {
-            throw new ServletException(ex);
-        }
 
-        if (td != null && td.length > 0) {
-            for (int i = 0; i < td.length; i++) {
-                if (td[i].getField("id").equals(strMWarehouseId)) {
-                    updateWarehouse = false;
-                    break;
-                }
+            if (warehouseIds.isEmpty()) {
+                info.addResult(WAREHOUSEID, "");
+                return;
```

Comment:
Stop callout execution? That's not right, the callout must complete its execution by setting the corresponding fields, it should never stop.